vcd/datasource_vcd_subscribed_catalog.go

```diff
@@ -0,0 +1,213 @@
+package vcd
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func datasourceVcdSubscribedCatalog() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdSubscribedCatalogRead,
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"storage_profile_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Optional storage profile ID",
+			},
+			"subscription_url": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The URL to subscribe to the external catalog. Required when 'subscription_catalog_href' is not provided",
+			},
+			"subscription_password": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "An optional password to access the catalog.",
+			},
+			"make_local_copy": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "If true, subscription to a catalog creates a local copy of all items. Defaults to false, which does not create a local copy of catalogItems unless sync operation is performed.",
```

Comment:
Probably this description is also from resource? As I see that it simply takes payload from VCD on read.

Reply:
yes. Updated the description